python/pyspark/ml/evaluation.py

```diff
@@ -15,13 +15,52 @@
 # limitations under the License.
 #
 
+from abc import abstractmethod, ABCMeta
+
 from pyspark.ml.wrapper import JavaEvaluator
 from pyspark.ml.param import Param, Params
 from pyspark.ml.param.shared import HasLabelCol, HasRawPredictionCol
 from pyspark.ml.util import keyword_only
 from pyspark.mllib.common import inherit_doc
 
-__all__ = ['BinaryClassificationEvaluator']
+__all__ = ['Evaluator', 'BinaryClassificationEvaluator']
+
+
+class Evaluator(Params):
```

Comment:
This is not needed because `Evaluator` doesn't override any methods from `Params`.